### ISSUE TITLE
US Band Plan Not Displaying Like Screenshots / Video

### ISSUE THREAD

**Issue by WareWolf-MoonWall** [Arrin-KN1E/SDRsharpbandplans#19]
Clean install of SDR# Studio V1.0.0.1889 on brand new laptop. Copied over the US band plan, and launched SDR#. Band plan has more detail than stock, but is thin line and lacking things like GMRS channels which I can see exist in the XML. 

Maybe just newbie error in setup, only been using SDR# a few days and barely know what I am doing yet.

**Comment by Arrin-KN1E** [comment]
@WareWolf-MoonWall I believe the newer version of SDR# displays the band plan differently now. The pictures and videos are from the older versions (This will be updated). There are a few options in the panel menu (3 horizontal lines; top left, 'Band Plan'), but it's still not the same. There may be a workaround for this, but that's out of my wheelhouse for now. 

The GMRS, CB, etc. allocations are channelized by default, so try zooming in to 462.600MHz/ 467.600MHz and you should see the channel numbers. If you don't like this default, you can use a text editor and comment out the channels and uncomment the more general allocation. If you need help on any of that let me know!

**Comment by WareWolf-MoonWall** [comment]
@Arrin-KN1E Makes sense that the software was updated. I've tried messing with it and can't get anything more than a thin line either along the bottom or top of the display with the "band" information at best. All the detail from within the XML file seems to be overlooked now and that was the entire point of grabbing this awesome module for the software, haha.

Have folks moved on from SDRSharp? I'm just getting into SDR and have only tried SDRSharp (I have the RTL-SDR dongle) and CubicSDR which I find harder to use.

**Comment by Arrin-KN1E** [comment]
@WareWolf-MoonWall Welcome to the hobby, it's never boring! 

If you really want it to look like the pictures, you could download and use an older version of SDR#. I personally use v1.0.0.1672 on my desktop (like the pictures) and v1.0.0.1855 on my laptop. If you have trouble finding an older version, let me know and I could send my version to you, to see if that works. 

I think one of the main issues people had/ have with SDR# is that for some people it's unstable. I've never had an issue with the many versions I've used over the years (sometimes even running over a month straight without issue). SDR# is still being maintained by Airspy, and from what I've seen plenty of people still use it (although a poll wouldn't be a bad idea). So, if you are comfortable with it and don't have any issues, there's no reason to not use it (unless you like something else of course). 

The biggest competitor to SDR# is [SDR++](https://github.com/AlexandreRouma/SDRPlusPlus). It's maintained more by the SDR community than SDR# is by Airspy, therefore SDR++ is more popular with the 'cutting edge' crowd, because anyone can make a module for it, so if that's what you are into, SDR++ is definitively better. I don't have Band Plan support for SDR++ yet as they use JSON rather than XML, but I plan on adding it somehow once I finish my new format.

**Comment by WareWolf-MoonWall** [comment]
@Arrin-KN1E Thanks for the suggestion on SDR++. I was able to get it setup on this laptop along with SDRSharp and CubicSDR. I'll play with all three as I continue to figure things out, but I'm liking SDR++ so far. _I wish I was further along in learning Rust so I could take a crack at an SDR app myself, but I have had too many projects._

As for your statement about XML to JSON, I know there are some free tools out there to handle just that, or you might be able to use a 30 day trial of Altova XMLSpy to do the conversion for you. Most of what I do for my day job is data transformation and systems integration. Hmmm...

**Comment by Arrin-KN1E** [comment]
@WareWolf-MoonWall Yeah, I know the pain of too many projects too. That's the reason I've unintentionally neglected this repository... What idea did you have, maybe someone over there could make it? 

Right now, the work on the next update is more about reformatting the comments in the file, and validating the allocations by providing the sources of those allocations. In fact, I think I'm going to split the update into parts, just to get something out there after all this time. So, before I do any conversions, or adapt other band plans, I want my main band plan to be creditable. However, if you have a simple script/ program that you want to try stuff with, I would be interested to see how it goes...
